### DISCUSSION TITLE
Small request, right arrow: changing -> to unicode 2192 (→)

### DISCUSSION THREAD

**Discussion by igtsimoes** [notaapp/talk#387] [Feature Requests]
Hello! 

I use the right arrow a lot. And it looks much nicer with arrow (→)than with -> .

Could we have this in a (not so far away) update?

Thank you!

**Comment by astoilkov** [comment]
Do you mean to transform -> into a → when you type it? Have you seen this somewhere? This feels more like a thing you would do with TextExpander, Alfred, or Raycast.

**Reply by igtsimoes** [reply]
Yes, this is "automatic" in Craft.do, Notion, Capacities.io and I guess in Obsidian also.

I have a text replacement setup in the keyboard settings (MacOS Ventura), but for some reason it doesn't work inside Nota. Works in Things and other apps though.

<img width="361" alt="image" src="https://user-images.githubusercontent.com/64813530/229955116-fb7e05af-21f4-4688-b711-f9da96cbabea.png">

**Reply by astoilkov** [reply]
Aha, got it. We will try to include support for text replacements in the next release.

**Comment by Strajk** [comment]
Related issue/discussion for reference https://github.com/notaapp/talk/discussions/89
